### PR TITLE
Portfolio project

### PR DESCRIPTION
**Pull request by gracekabaghe**
This is my portfolio showing both my soft and technical skill-sets.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    <div class="container">
+        <div class="navbar">
+            <div class="logo">
+                G
+            </div>
+            <div class="menu">
+                <i class="fas fa-bars"></i>
+            </div>
+        </div>
```
Comment (Mihndim2020) [inline index.html:24]: - [ ] The container of this section should be a `<header>` tag. Please kindly fix that as it is part of the project requirements. This also makes your html more semantic, which is good for assistive technologies, and search engine optimization. Please checkout this [link](https://www.w3schools.com/html/html5_semantic_elements.asp) about semantic html.
- [ ] Please also make sure you insert the proper background images as in the design. 
- [ ] Please make sure this section has the margins (top, bottom, left, right) defined in the design
- [ ] Please use the fonts and colors (background and font) defined for the template

     ## Your implementation ##    
grace

 ## The design ##     
thedesign

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;0,700;1,400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <div class="container">
+        <div class="navbar">
+            <div class="logo">
+                G
```
Comment (Mihndim2020) [inline index.html:19]: - [ ] The logo should be link tag. Please kindly fix that. If you check on most websites, you will notice that the logo is a link the takes you to the home page, enhancing the site navigation.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            </div>
+            <div class="menu">
+                <i class="fas fa-bars"></i>
+            </div>
```
Comment (Mihndim2020) [inline index.html:23]: - [ ] The hamburger button should be within a `<nav>` tag, as this is part of the project requirements. Please fix that.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        </div>
+        <div class="section">
```
Comment (Mihndim2020) [inline index.html:25]: - [ ] The container of this section should be a `<section>` tag as per requirements and for semantic purposes as explained above. Please kindly fix that.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            <div class="hero">
+                <p>Hey! <br>
```
Comment (Mihndim2020) [inline index.html:27]: - [ ] The main title is an `<h1>` tag. Please kindly fix that. 
- [ ] The content should be aligned with flexbox.
- [ ] The section should have margins (top, bottom, left, right) defined in the design.
- [ ] The section should use the font and colors (background and font) defined for the template. Please kindly fix that.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        <div class="connect">
+            <div class="link">
+                <a href="#">Reach Out</a>
+            </div>
+            <div class="social">
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-instagram"></i>
+                <i class="fab fa-facebook"></i>
+                <i class="fab fa-linkedin"></i>
+            </div>
```
Comment (Mihndim2020) [inline index.html:46]: - [ ] The list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag). Please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @gracekabaghe, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Linters are passing
:heavy_check_mark: The project is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION



# Status: Require Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please also consider:
- [OPTIONAL] using imperative tense in your commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Leon-Mbegera — APPROVED** [review]
Hi @gracekabaghe , 

# Approved 🎉 

- Well done on implementing all the requested changes by the previous reviewer 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it and proceed onto the next stage. :shipit: 
Congratulations! 🎉 

## Optional 
- [ ] Beware this is not a required change but a suggestion. The previous reviewer missed to point out that your social media icons need some margins in between each other as per the given design. It would be nice to fix that. However, your project is still approved 😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**